lib/stagehand/staging/synchronizer.rb

```diff
@@ -7,7 +7,11 @@ module Synchronizer
       BATCH_SIZE = 1000
       SESSION_BATCH_SIZE = 30
       ENTRY_SYNC_ORDER = [:delete, :update, :insert].freeze
-      ENTRY_SYNC_ORDER_SQL = ActiveRecord::Base.send(:sanitize_sql_for_order, ['FIELD(operation, ?), id DESC', ENTRY_SYNC_ORDER]).freeze
+      ENTRY_SYNC_ORDER_SQL =  begin
+                                ActiveRecord::Base.send(:sanitize_sql_for_order, ['FIELD(operation, ?), id DESC', ENTRY_SYNC_ORDER]).freeze
```

Comment:
Here all we need to do is use Arel instead to avoid sanitizing at load time.

Reply:
What exactly are we sanitizing here? Is the whole expression not equal to:

`"FIELD(operation, 'delete','update','insert'), id DESC"`

? There is not dynamic input where sanitization would be necessary, or am I overlooking something?

Reply:
Btw: regarding Rails' "wrap this in Arel.sql()" deprecation warning, I've opened this PR: https://github.com/combinaut/sparkle/pull/12998